examples/vfts243.jl

```diff
@@ -1,21 +1,72 @@
 using SideKicks
-using Turing
 using BenchmarkTools
-##
+using CairoMakie
+using Distributions
+
+obs = SideKicks.createObservations([
+    [:P,   10.4031,  0.01,   day],
+    [:e,   0.017,    0.012,  1],
+    [:m1,  25.0,     2.3,    m_sun],
+    [:K1,  81.4,     1.3,    km_per_s],
+]) 
 
-observed_values2 = [10.4031, 0.017, 25.0, 81.4]
-observed_errors2 = [0.01, 0.012, 2.3, 1.3]
-obs2 = [:P, :e, :m1, :K1]
+priors = SideKicks.createPriors(
+    logm1_dist = Uniform(0.1,3), # in log(Msun)
+    logm2_dist = Uniform(0.1,3), # in log(Msun)
+    logP_dist  = Uniform(-1,3),   # in log(days)
+    vkick_dist = Exponential(1), # in 100 km/s
+    frac_dist  = Uniform(0,1.0)
+)
 
 ##
-example_model2 = SideKicks.createSimpleCircularMCMCModel(obs2, observed_values2, observed_errors2);
-iterations = 2_000
-@time chain2 = sample(example_model2, NUTS(5_000,0.8), MCMCThreads(), iterations, 8);
+
+mcmcStruct = SideKicks.RunKickMCMC(
```

Comment:
do not like adding Struct to object names. Why not just call this `mcmc` or `mcmc_output`?  Also, if we care about code consistency we have a bit of a mixture of camel case and snake case. I generally prefer camel case for structs, constructors and modules, snake case for everything else. But adjusting coding guidelines can be dealt with later on, perhaps make an issue for it.

Reply:
Agreed that we need to make the case choices more consistent, I'm happy to default to your preference. As for naming things, I'm trying to make it all as obvious as possible for new users (like me), who will get confused by something just named `mcmc` or `mcmc_output`. The struct suffix means they will look to the struct definition to know what's contained in this object.

Reply:
I still do not think you need to do that. You don't really know what struct it is with that information. What is needed in that case is to have good documentation, so people can realize what is this return object. And when working interactively, you can always check the type directly.